optimize/self-managed/optimize-deployment/migration-update/translation-diffs/differences_localization_312_313.diff

```diff
@@ -0,0 +1,25 @@
+diff --git a/backend/src/main/resources/localization/en.json b/backend/src/main/resources/localization/en.json
+index 659a884148..12887fd12d 100644
+--- a/backend/src/main/resources/localization/en.json
++++ b/backend/src/main/resources/localization/en.json
+@@ -27,9 +27,7 @@
+     "termsOfUse": "Terms of use",
+     "imprint": "Imprint",
+     "academy": "Camunda Academy",
+-    "feedback": "Feedback and Support",
```

Comment:
this diff looks like it was generated the other way around, cause the `---` changes look like the new ones and `+++` like the old